openslides_backend/permissions/permissions.py

```diff
@@ -47,21 +48,21 @@ def __str__(self) -> str:
 
 
 class _AgendaItem(Permission, Enum):
-    CAN_SEE_INTERNAL = "agenda_item.can_see_internal"
```

Comment:
It's kind of annoying that the order seems to not be consistent here. Do you mind sorting the permissions alphabetically in the generation file? Changing `cli/generate_permissions.py#L97` (in this PR) to `for permission in sorted(permissions):` should suffice for that, I think.

Reply:
Done